ClassMoodApp/Models/API.py

```diff
@@ -10,13 +10,17 @@ class API(object):
     @staticmethod
     def is_student(user_id):
         user = DBModels.User.query.filter_by(id=user_id).first()
-        user_type_obj = DBModels.UserType.query.filter_by(id=user.user_type).first()
-        return user_type_obj.name == STUDENT
+        if user:
+            return user.is_student
+        return False
+
     @staticmethod
     def is_professor(user_id):
         user = DBModels.User.query.filter_by(id=user_id).first()
-        user_type_obj = DBModels.UserType.query.filter_by(id=user.user_type).first()
-        return user_type_obj.name == PROFESSOR
+        if user:
+            return ~user.is_student
```

Comment:
'~' is a bitwise operator, and will return a number instead of a boolean.  Use 'not' instead